example/data/acceptable_tekton_tasks_git.yml

```diff
@@ -0,0 +1,64 @@
+---
+# This file is meant to illustrate the data structure used to specify the list of acceptable git
+# commits for Task definitions consumed via Tekton's git resolver
+# (https://tekton.dev/docs/pipelines/git-resolver/)
+
+# Closely align with the current format of acceptable bundles.
+acceptable-task-git-refs-option1:
+  https://github.com/redhat-appstudio/build-definitions.git:
+    task/buildah/0.1/buildah.yaml:
+      - commit: 3672a457e3e89c0591369f609eba727b8e84108f
+        effective_on: 2024-02-16T00:00:00Z
+      - commit: 81185d669f543f5cd9114377f7ee0480e15a7ac1
+        effective_on: 2024-02-10T00:00:00Z
+    task/clair-scan/0.1/clair-scan.yaml:
+      - commit: 28b86c8b94c42995d98f843b0fc854c1cf2ae500
+        effective_on: 2024-02-15T00:00:00Z
+      - commit: 4efdfc8c8da9f78602b5a21bcd2b3cf5667503fc
+        effective_on: 2024-01-31T00:00:00Z
+
+# Align with the future of acceptable bundles format:
+# https://github.com/enterprise-contract/ec-cli/issues/1291
+acceptable-task-git-refs-option2:
+  https://github.com/redhat-appstudio/build-definitions.git//task/buildah/0.1/buildah.yaml:
+    - commit: 3672a457e3e89c0591369f609eba727b8e84108f
+      effective_on: 2024-02-16T00:00:00Z
+    - commit: 81185d669f543f5cd9114377f7ee0480e15a7ac1
+      effective_on: 2024-02-10T00:00:00Z
+  https://github.com/redhat-appstudio/build-definitions.git//task/clair-scan/0.1/clair-scan.yaml:
+    - commit: 28b86c8b94c42995d98f843b0fc854c1cf2ae500
+      effective_on: 2024-02-15T00:00:00Z
+    - commit: 4efdfc8c8da9f78602b5a21bcd2b3cf5667503fc
+      effective_on: 2024-01-31T00:00:00Z
+
+# Align with how the git resolver is used.
```

Comment:
If we set the base model on the Tekton bundle resolvers, i.e. we don't support the legacy bundle. Is there a way to encode both git and oci/bundle Task references in the same structure?

Reply:
I think so. There's value in, somehow, combining those two pieces of information. Added option 5 to the bottom (which is a variation of option 2).